packages/main/src/themes/CheckBox.css

```diff
@@ -53,7 +53,7 @@
 }
 
 :host([wrapping-type="Normal"][text]) .ui5-checkbox-root .ui5-checkbox-label {
-	word-break: break-all;
+	overflow-wrap: break-word;
```

Comment:
**[praise (non-blocking)](https://conventionalcomments.org/#labels):** https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap#browser_compatibility and https://caniuse.com/?search=overflow-wrap show that `overflow-wrap: break-word` has been supported by all the modern browsers for some time now, so that's good.